client/state/selectors/test/get-site-url.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getSiteUrl } from '..';
+
+describe( 'getSiteUrl()', () => {
+	it( 'should return null if the site is unknown', () => {
+		const state = {
+			sites: {
+				items: {}
```

Comment:
It's usually better to have a site with another ID when testing that case - this helps verify that the selector is not selecting wrong data.